tests/config/lineEndings-to-Mac.cfg

```diff
@@ -0,0 +1,19 @@
+indent_columns   = 3
+indent_with_tabs = 0
+# (284)Align continued statements at the '='. Default=True
```

Comment:
Do those comment have a meaning? I know that the text is the option description but what about the numbers?

Reply:
any approvment?